suite-native/app/package.json

```diff
@@ -112,6 +111,7 @@
         "react-native-gesture-handler": "2.18.1",
         "react-native-keyboard-aware-scroll-view": "0.9.5",
         "react-native-mmkv": "2.12.2",
+        "react-native-quick-crypto": "0.7.6",
```

Comment:
you probably want to change runtimeVersion in app.config.js because of preview build

Reply:
Yea, expo dev client crash now

Reply:
Incremented.